### PR TITLE
chuyen ve query ve repository

### PR DESCRIPTION
**Pull request by kienpham2004**
## Related Tickets:
- [#33301(https://edu-redmine.sun-asterisk.vn/issues/33301)

## WHAT this PR does?
Chuyển controller Follow, Like, Admin về repository

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by kienpham2004** [comment]
ready

File: app/Repositories/Role/RoleRepository.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?php
+namespace App\Repositories\Role;
+
+use App\Repositories\BaseRepository;
+use App\Models\Role;
+use Illuminate\Support\Facades\Auth;
```
Comment (huyenttm-0901) [inline app/Repositories/Role/RoleRepository.php]: cái Auth có cần ko em?

File: resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -31,7 +31,7 @@
         crossorigin="anonymous">
     <script src="{{ asset('js/search.js') }}"></script>
     <script src="{{ asset('js/follow.js') }}"></script>
-    <script src="{{ asset('js/like_post.js') }}"></script>
+    {{-- <script src="{{ asset('js/like_post.js') }}"></script> --}}
```
Comment (huyenttm-0901) [inline resources/views/layouts/app.blade.php]: ko dùng xoá đi nhé

File: app/Repositories/Role/RoleRepository.php (hunk 1 above)
Comment (kienpham2004) [reply app/Repositories/Role/RoleRepository.php]: không ạ

**Comment by kienpham2004** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**